changelog.txt

```diff
@@ -1,3 +1,21 @@
+4.8.1.
```

Comment:
Are we not looking at fixing https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1487 before releasing?

Reply:
i'm looking at it now